SensioLabs/Security/Crawler/BaseCrawler.php

```diff
@@ -20,6 +20,7 @@ abstract class BaseCrawler implements CrawlerInterface
 {
     protected $endPoint = 'https://security.sensiolabs.org/check_lock';
     protected $timeout = 20;
+    protected $excludedCVEs = array();
```

Comment:
I think it is better to use `[]`, as that is what I see in other classes as well.